[NL] Huishoudelijk_Reglement.md

```diff
@@ -298,4 +298,4 @@ Voor niet begrote uitgaven die tijdens een bestuursperiode samen meer dan het mi
 
 ### Artikel 52
 
-De Nederlandse versie van dit huishoudelijk regelement is leidend.
\ No newline at end of file
+De Nederlandse versie van dit huishoudelijk reglement is leidend.
```

Comment:
@mvr320 is changing this allowed you think?

Reply:
Thats something for the ALV to decide, I think in general the VC will accept spelling changes, whereas grammar could change the intended meaning so might be a bit wrong?

Another system which I would prefer is to do the following:
If one wants to change a HR rule (which could be a motion) (s)he creates a PR on GitHub before the GA so there is a good wording (well thought of) and people can already form an opinion. After the GA the board will merge this change if/when its accepted at the GA (maybe with a link to the minutes with the discussion on this new or altered rule). With these rules you could decide to wait till the next GA (Which is fine by me.. you could also close this if you want)
This would help you to have a DB of motions (which is a task for the board and is always forgotten) and things would endup in the HR (which I think is a better location compared to having a long lists of motions which you only understand when you read about the discussion)

But... I'll not be at GAs so I leave this up to the current board to decide how they want to handle this. So feel free to just close it if you don't have an opinion yet.